### DISCUSSION TITLE
[SDK] <Title>

### DISCUSSION THREAD

**Discussion by JabariKun** [zkSync-Community-Hub/zksync-developers#234] [SDKs] [answered]
### Environment

Mainnet

### Acknowledgement

- [X] I understand that issues are expected with the Go SDK as they are being updated.

### Issue Description

There is no java SDK dependency code in the document

### Expected Behavior

There is no java SDK dependency code in the document

### Code Example

There is no java SDK dependency code in the document

**Comment by dutterbutter** [comment, marked as answer]
Closing as there is no question asked. Please open a new discussion with relevant details.
